desktop/appcontainer/ExceptionDialogDetails.js

```diff
@@ -29,7 +30,7 @@ export const exceptionDialogDetails = hoistCmp.factory(
 
         if (!detailsIsOpen || !exception) return null;
 
-        const errorStr = stringifyErrorSafely(exception);
+        const errorStr = stringifyErrorSafely(exception, ExceptionHandler.REDACT_PATHS);
```

Comment:
why not just build this in to stringifyErrorSafely?  Seems to be totally appropriate.

Reply:
I wanted to, but you get a circular import between Stringify.js and ExceptionHandler.js that breaks the app

Reply:
We could put the overridable static property on the Exception class to get around it?

Reply:
we can declare the list in Stringify.js, was not sure it needed to be on ExceptionHandler anyway.